pkg/splunk/enterprise/standalone.go

```diff
@@ -123,6 +123,35 @@ func ApplyStandalone(client splcommon.ControllerClient, cr *enterprisev1.Standal
 		return result, err
 	}
 
+	// If we are using appFramework and are scaling up, we should re-populate the
+	// configMap with all the appSource entries. This is done so that the new pods
+	// that come up now will have the complete list of all the apps and then can
+	// download and install all the apps.
+	// TODO: Improve this logic so that we only recycle the new pod/replica
+	// and not all the existing pods.
+	if len(cr.Spec.AppFrameworkConfig.AppSources) != 0 && cr.Spec.Replicas > 1 {
+
+		statefulsetName := GetSplunkStatefulsetName(SplunkStandalone, cr.GetName())
+
+		ok, err := splctrl.IsStatefulSetScalingUp(client, cr, statefulsetName, cr.Spec.Replicas)
```

Comment:
nit: instead of ok, you may want to use a more logical flag name for better clarity :-)